kostyor_cli/main.py

```diff
@@ -26,18 +26,34 @@ def _make_request_with_cluster_id(http_method, endpoint, cluster_id):
 
 
 def _print_error_msg(resp):
-    message = resp.json()['message']
+    try:
+        message = resp.json()['message']
+    except (ValueError, KeyError):
+        # if response doesn't contain valid error representation
+        # just print its content
+        message = resp.text
+
     print('HTTP {}: {}'.format(resp.status_code, message))
 
 
 class KostyorApp(App):
+
     def __init__(self):
         super(KostyorApp, self).__init__(
             description='Kostyor cli app',
             version='0.1',
             command_manager=CommandManager('kostyor.cli'),
             deferred_help=True,
-            )
+        )
+
+        # There are two reasons why using session is a good idea here. The
+        # first one is, session uses a pool with keep-alive connections so
+        # any further requests to the same domain won't initiate a new
+        # connection. The second reason is, it's configurable so we can
+        # setup, for example, common headers here and do not pass them
+        # across the code.
+        self.request = requests.Session()
+        self.baseurl = 'http://{0}:{1}'.format(host, port)
```

Comment:
It can still be part of this pull request, I just want you to break it out into its own commit